src/m365/spo/commands/list/list-remove.ts

```diff
@@ -57,6 +59,9 @@ class SpoListRemoveCommand extends SpoCommand {
       {
         option: '-t, --title [title]'
       },
+      {
+        option: '--recycle'
```

Comment:
Could you please also add an initTypes function and force type all options?

Reply:
added